src/highlight/src/extension.ts

```diff
@@ -416,6 +416,7 @@ function updateRainbowBrackets() {
     }
   }
   activeEditor.setDecorations(misplacedType, misplaced);
+  mirrorDoc["highlightFoundMisplacedBrackets"] = misplaced.length; // help tests 'see' that misplaced brackets are highlighted
```

Comment:
Assuming this property will be used elsewhere (in tests?), I think it's a good idea to assign it to a var somewhere and only use the string in that assignment, and use the var here and elsewhere.